src/plugins/expressions/common/service/expressions_services.ts

```diff
@@ -290,6 +290,26 @@ export class ExpressionsService implements PersistableState<ExpressionAstExpress
     return this.executor.inject(state, references);
   };
 
+  /**
+   * Injects saved object references into expression AST
+   * @param state expression AST to update
+   * @param references array of saved object references
+   * @returns new expression AST with references injected
+   */
+  public readonly migrate = (state: SerializableState, version: string) => {
+    return this.executor.migrate(state, version);
+  };
+
+  /**
+   * Injects saved object references into expression AST
+   * @param state expression AST to update
+   * @param references array of saved object references
+   * @returns new expression AST with references injected
+   */
+  public readonly migrateToLatest = (state: unknown, version: string) => {
+    return this.executor.migrateToLatest(state, version);
+  };
```

Comment:
Code comments on both of these functions were copied from `inject` but still need to be updated with the correct jsdoc.

Reply:
yeah this is still not complete, if you look at PR description you'll see docs and tests are still missing

Reply:
oops, missed that in the checklist 👍